### PR TITLE
lots of small stuff

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Removes: 
pilgrim claw
pilgrim hero
SST LMG
SST second handgun

SSTs no longer shrink commically down and unevenly

Fixes void reavers dropping .223 mags and cases

Bookshelfs now hold papers, folders and bundles of papers
Bookshelfs have been added to cargo areas

Rewrights the pilgrims descs
pilgrim devout now has slight less recoil control and a slower rate of fire do to balance and the belt feeding
Guild venders now are properly linked to guild accounts
Guild venders stock has been greatly improved
Medical venders stock has been greatly improved

Opfex tool belt now has the proper hammer

Sec belts hold more types of small guns, but no longer holds pilgrims
Pistol Holsters that go on belt and pockets now hold a large selection of pistols, laser guns and sawn weaponery

### CODE REVIEW

File: code/game/objects/items/weapons/storage/pouches.dm [hunk 1]
```diff
@@ -322,20 +323,60 @@
...
+		/obj/item/gun/energy/laser/railgun/pistol,
+		/obj/item/gun/energy/laser/makeshift,
+		/obj/item/gun/energy/cog/sawn,
 		/obj/item/gun/projectile/revolver,
+		/obj/item/gun/projectile/revolver/sixshot/sawn,
 		/obj/item/gun/projectile/shotgun/doublebarrel/sawn, //short enough to fit in
 		/obj/item/gun/launcher/syringe,
 		/obj/item/gun/energy/plasma/auretian,
-		/obj/item/gun/energy/centurio,
-		/obj/item/gun/projectile/boltgun/flare_gun
```
Comment (faaaay) [inline code/game/objects/items/weapons/storage/pouches.dm:336]: Why did you remove flare guns from the list?

Comment (Trilbyspaceclone) [reply code/game/objects/items/weapons/storage/pouches.dm:336]: I didnt its moved down

Comment (faaaay) [reply code/game/objects/items/weapons/storage/pouches.dm:336]: oh my bad
